setup.py

```diff
@@ -12,18 +12,22 @@
     long_description=readme,
     long_description_content_type='text/markdown',
     name="tableau_utilities",
-    version="2.0.0",
+    version="2.0.1",
     packages=[
         'tableau_utilities',
         'tableau_utilities.general',
         'tableau_utilities.tableau_file',
-        'tableau_utilities.tableau_server'
+        'tableau_utilities.tableau_server',
+        'tableau_utilities.scripts'
     ],
     package_data={'tableau_utilities': ['tableau_file/*.yml']},
     include_package_data=True,
     entry_points={
         'console_scripts': [
-            'tableau_utilities = tableau_utilities.main:main'
+            'tableau_utilities = tableau_utilities.scripts.main:main',
+            'tab-util-config-gen = tableau_utilities.scripts.datasources_column_config_generate:main',
+            'tab-util-config-merge = tableau_utilities.scripts.datasources_column_config_merge:main',
+            'tab-util-server = tableau_utilities.scripts.server:main',
```

Comment:
Not sure if we can do it like this, but it might be nice to be able to call the scripts like this:
```
tableau_utilities --generate_config
tableau_utilities --merge_config
```

Reply:
@JustinGrilli  I got that working with subparsers :)